Cargo.toml

```
@@ -42,7 +42,7 @@ name = "daft"
 edition = "2021"
 name = "daft"
 publish = false
-version = "0.1.10"
+version = "0.2.0+dev0"
```

Nit: Should be a `-` separator instead of a `+` to match semver for pre-release versions; `+` is a separator for build metadata.
```suggestion
version = "0.2.0-dev0"
```

---

I was using "+" before since the hypen would make maturin crap out when building a python wheel but it looks like it works now!